### ISSUE TITLE
Project roadmap?

### ISSUE THREAD

**Issue by aatkinson** [facebookresearch/pytext#254]
Hi,

The goals are ambitious, the codebase is in flux, and future directions are outlined in your paper https://arxiv.org/pdf/1812.08729.pdf, but it's unclear what the roadmap is for this project.

What release cadence can we expect, and what features are prioritized?

I'm also curious if this has a substantial backing by Facebook / the PyTorch team.

Thanks

**Comment by ahhegazy** [comment]
Great questions :)

As for the road map, we are mainly working on 5 main areas (roughly sorted according to priority):

- Enriching and Improving our stack with more tasks, models and techniques
- Improve Usability and have better integration with ipython notebooks
- Performance optimizations for training and inference
- Explore model interpretability techniques

For the release cadence, we will do our best to have a release every month. And yeah as we mentioned in our release blog post [here](https://fburl.com/oqd0nim4) this project powers some of the core projects  in production at Facebook and it indeed has a substantial backing from different teams in Facebook AI.

**Comment by aatkinson** [comment]
Neat, this is all good to know, thanks Ahmed.

I'm looking forward to seeing more model and task examples, particularly advanced ones.

Also that release blog post link appears to be broken for me, or at least inaccessible. I'm assuming it's the same as this one? https://code.fb.com/ai-research/pytext-open-source-nlp-framework/.

**Comment by ahhegazy** [comment]
Yes, this is the right link.

**Comment by maxdel** [comment]
Thanks for the answer @ahhegazy! Does integration with fairseq fall under the 1st area?

**Comment by ibulu** [comment]
> Thanks for the answer @ahhegazy! Does integration with fairseq fall under the 1st area?

I am curious about the fairest integration as well. Any plans on that?

**Comment by ahhegazy** [comment]
Yes, we are working on integrating with translate: https://github.com/pytorch/translate/tree/master/pytorch_translate which is Fairseq models with production support

**Comment by maxdel** [comment]
@ahhegazy Another quick question, does multilingual modeling support is still on the radar? 
Also really interested regarding fairseq timing question. 

And thanks for sharing this codebase with the community!
